runtime/parachains/src/inclusion.rs

```diff
@@ -2410,6 +2470,56 @@ mod tests {
 				vec![CoreIndex::from(0), CoreIndex::from(1), CoreIndex::from(2)]
 			);
 
+			// Transform the votes into the setup we expect
+			let expected = {
+				let mut intermediate = std::collections::HashMap::<
```

Comment:
In the runtime, we should use `BTreeMap` as randomness is not properly available for hash maps and this can lead to attacks. It can also lead to mismatches in consensus.

Reply:
(ah, this is test code)